puddles/Backplane/src/dshot.c

```diff
@@ -150,11 +150,15 @@ void __time_critical_func(dshot_telem_cb)(PIO pio_hw) {
 }
 
 void __time_critical_func(dshot_telem_pio0_cb)(void) {
+    // profiler_push(PROFILER_DSHOT_RX_IRQ0);
```

Comment:
Re-enable profiler?

Reply:
I think i can re-enable these now. This was from hunting the bad profiler issue from earlier